Tools/LambdaTestTool/src/Amazon.Lambda.TestTool.BlazorTester/Startup.cs

```diff
@@ -95,16 +101,70 @@ public void ConfigureServices(IServiceCollection services)
         }
 
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
-        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ILamdaService lamdaService, LocalLambdaOptions localLambdaOptions, IAwsProfileConfig awsProfileConfig)
         {
             app.UseDeveloperExceptionPage();
 
             app.UseStaticFiles();
 
             app.UseRouting();
 
+            app.UseCors();
+            var configInfo = awsProfileConfig.LambdaConfigInfo();
+
             app.UseEndpoints(endpoints =>
             {
+                if (configInfo != null)
```

Comment:
If the fields change in the UI like the chosen config file the mapped endpoints won't get updated. I wonder if it would be better to inject a middleware that could be more dynamic and look to see if the incoming request matches any functions currently available based on the current UI settings. If so the invoke the Lambda function otherwise return continue the middleware pipeline.